From ecf397760bfe1a5f9742cafa75761e68678c38ee Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 28 Feb 2017 16:13:18 +0100
Subject: [PATCH] change of rtnetlink to use RTN_F_OFFLOAD

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1417289
Upstream Status: iproute2.git commit c52827e9077f7

commit c52827e9077f781f4d5f52f0c0aa67b59151e187
Author: Stephen Hemminger <shemming@brocade.com>
Date:   Wed May 27 18:29:02 2015 -0700

    change of rtnetlink to use RTN_F_OFFLOAD

    The definition of offload flag changed during 4.1 rc process.
---
 ip/iproute.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ip/iproute.c b/ip/iproute.c
index 73c6db9..eb5d145 100644
--- a/ip/iproute.c
+++ b/ip/iproute.c
@@ -438,7 +438,7 @@ int print_route(const struct sockaddr_nl *who, struct nlmsghdr *n, void *arg)
 		fprintf(fp, "onlink ");
 	if (r->rtm_flags & RTNH_F_PERVASIVE)
 		fprintf(fp, "pervasive ");
-	if (r->rtm_flags & RTNH_F_EXTERNAL)
+	if (r->rtm_flags & RTNH_F_OFFLOAD)
 		fprintf(fp, "offload ");
 	if (r->rtm_flags & RTM_F_NOTIFY)
 		fprintf(fp, "notify ");
-- 
1.8.3.1

